mirror-wordpress-plugins

```diff
@@ -37,23 +37,23 @@ class PluginUpdater
   end
 
   def fetch_api_information
-    @latest_github_version = fetch_latest_github_version
-    @latest_wordpress_version, @wordpress_download_link = fetch_latest_wordpress_info_and_annotate_repo
-    @zip_file = "#{@full_path_to_clone}-#{@latest_wordpress_version}.zip"
+    @latest_plugin_version_on_github = fetch_latest_plugin_version_on_github
+    @latest_plugin_version_on_wordpress, @wordpress_download_link = fetch_latest_wordpress_info_and_annotate_repo
+    @zip_file = "#{@full_path_to_clone}-#{@latest_plugin_version_on_wordpress}.zip"
   end
 
   def repos_need_updating?
-    puts "==> #{@slug} is #{@latest_wordpress_version} on WordPress and #{@latest_github_version} on GitHub"
-    if @latest_github_version === "" || @latest_github_version === "v"
+    puts "==> #{@slug} is #{@latest_plugin_version_on_wordpress} on WordPress and #{@latest_plugin_version_on_github} on GitHub"
```

Comment:
I'm not quite sure why, but I'm getting a linebreak before `on GitHub` here that I wasn't getting before.

e.g. on `main`:
```sh
==> seo-by-rank-math is v1.0.230 on WordPress and v1.0.230 on GitHub
```

on this branch:
```sh
==> seo-by-rank-math is v1.0.230 on WordPress and v1.0.230
 on GitHub
```